src/cljs/main/broadfcui/page/workspace/data/import_data.cljs

```diff
@@ -47,21 +48,21 @@
       (if-let [result (:upload-result @state)]
         (if (:success? result)
           (style/create-flexbox {:style {:justifyContent "center" :paddingTop "1em"}}
-            (icons/icon {:style {:fontSize "200%" :color (:success-state style/colors)}} :done)
-            [:span {:style {:marginLeft "1em"}} "Success!"])
+                                (icons/icon {:style {:fontSize "200%" :color (:success-state style/colors)}} :done)
+                                [:span {:style {:marginLeft "1em"}} "Success!"])
```

Comment:
I had specifically aligned this "wrong" on purpose.  However, I don't want to fight with my editor.  Let's have IntelliJ indent this differently, and put that formatting into the master format.xml (or whatever that thing is).